Docs: fixed descriptions (#3125)
juicedata/juicefs: docs/en/introduction/architecture.md

title: Architecture
sidebar_position: 2
slug: /architecture

> description: This article introduces the technical architecture of JuiceFS and the resulting technical advantages, as well as the file storage principles of JuiceFS.

Reviewer: ```suggestion
description: This article introduces the technical architecture of JuiceFS and its technical advantages.
```
